cli/src/args.ts

```diff
@@ -77,10 +85,34 @@ export const args: BenchmarkArguments = {
       task.trainingInformation.batchSize = unsafeArgs.batchSize;
       task.trainingInformation.roundDuration = unsafeArgs.roundDuration;
       task.trainingInformation.epochs = unsafeArgs.epochs;
+      task.trainingInformation.validationSplit = unsafeArgs.validationSplit;
 
       // For DP
-      // TASK.trainingInformation.clippingRadius = 10000000
-      // TASK.trainingInformation.noiseScale = 0
+      const {dpDefaultClippingRadius, noiseScale, delta} = unsafeArgs;
+
+      if (
+        dpDefaultClippingRadius !== undefined ||
+        noiseScale !== undefined ||
+        delta !== undefined
+      ){
+        if(
+          task.trainingInformation.scheme === "decentralized" ||
+          task.trainingInformation.scheme === "federated"
+        ){
```

Comment:
this will silently discard user privacy settings if other schemes, better to throw

```suggestion
        if (task.trainingInformation.scheme === "local")
          throw new Error("can't have differential privacy for local training");
```